### PR TITLE
Add spatial_batch_norm option to resnet 50 model

### PR DESCRIPTION
**Pull request by davelgraphcore**
Sometimes when benchmarking/testing it is useful to build a resnet model
without batch normalization. Although an option already existed it did
not pass up to the create_resnet50 function.

### CODE REVIEW

**Comment by Yangqing** [comment]
@akyrola Aapo, do you mind taking a look after CLA signed?

**Comment by orionr** [comment]
Thank you for your contribution! PyTorch and Caffe2 are now officially merged with the PyTorch 1.0 preview release. Please submit on https://github.com/pytorch/pytorch if this is still an issue over there.
